### PR TITLE
Added new test for find bright pixels (error pixels) of a lcd screen.

### PR DESCRIPTION
**Pull request by ThKattanek**
Hi! I have a new test added. This test is for finding error pixels from a LCD monitor. Cycle color red -> green -> blue -> white -> black.

### CODE REVIEW

File: screentest.ui [hunk 1]
```diff
@@ -157,6 +157,16 @@
                 <signal name="activate" handler="on_mode_change"/>
               </object>
             </child>
+            <child>
+              <object class="GtkRadioMenuItem" id="bright_pixels">
+                <property name="visible">True</property>
+                <property name="label" translatable="yes">Bright Pixels</property>
```
Comment (TobiX) [inline screentest.ui:163]: I wonder if "Solid color" would be the better label? Not sure about this myself :smile:

File: test_basic.c [hunk 2]
```diff
@@ -67,11 +68,12 @@ static void basic_draw(GtkWidget * widget)
 	gint w, h;
 	gint i, b, d;
 	gint maxwidth, maxheight;
-	gint widths[6];
+        gint widths[7];
```
Comment (TobiX) [inline test_basic.c]: Formatting is currently TABs throughout the project. Could you try to keep the formatting consistent?

**Review by TobiX — CHANGES_REQUESTED** [review]
Looks good so far, just some minor nitpicks.

File: test_basic.c (hunk 2 above)
Comment (ThKattanek) [reply test_basic.c]: Ohhh sorry. Yes it is no problem. Qr Creator has filled the tab with space at me! I have this changed in the settings. :)

File: screentest.ui (hunk 1 above)
Comment (ThKattanek) [reply screentest.ui:163]: The Name is no matter. "Solid Color Cycle RGBWB". What ever ;)

**Pull request merged**